frontend_app_logbook/src/css/index.styl

```diff
@@ -0,0 +1,341 @@
+              &::after
+                display none
+          & button:hover
+            background-color lightGrey1
+          &.buttonHidden
+            .dropdown, .logbook_addCard
+              display none
+          &.readOnly
+            strong
+              cursor auto
+
+        & .react-kanban-board
+          direction ltr
+          height 100%
+          overflow-x visible
+          padding standardSpacing
+          & > div
+            height 100%
+
+      &__toolbar
+        position absolute
+        top standardSpacing
+        z-index 1
+        & .iconbutton
+          background-color offWhite
+          &:hover
+            background-color lightGrey2
+        &:empty
+          display none
+
+      & .react-kanban-column-header
+        display flex
+        flex-direction row
+        > :first-child
+          flex 1
+        > :last-child
+          display inline-block
+          border-radius standardBorderRadius
+          cursor default
+
+      & .react-kanban-column
+        padding standardSpacing
+        background-color lightGrey2
+        border-radius standardBorderRadius
+        margin 0 textSpacing
+        min-width 315px
+        text-align center
+        & > :last-child
+          overflow hidden
+          // INFO - GB - 2021-12-20 - The css rule below is !important because it overrides the library style
+          height calc(100% - 43px) !important
+          &:hover
+            overflow-y auto
+  &__LogbookPopup
+    max-height 80%
+    &.hidden
+      display none
+    &__bgColor, &__deadline
+      display flex
+      flex-direction column
+      &__inline
+        flex-direction row
+
+    & .cardPopup__container
+      width 80%
+      display flex
+      flex-direction column
+      overflow auto
+
+    & [type=checkbox]
+      margin textSpacing
+
+    & [type=text], textarea
+      width 100%
+
+    & [type=color]
+      margin-bottom textSpacing
+
+    & label
+      white-space pre
+      font-weight bold
+
+    & input
+      vertical-align middle
+
+    &__description, &__deadline, &__title, &__freeInput
+      margin-bottom standardSpacing
+
+    &__description
+      > div
+        height 200px
+
+    &__form
+      overflow-y auto
+      width 100%
+      height 100%
+      &__fields
+        width calc(100% - 2px) // NOTE - RJ - 19-01-2022 - TinyMCE borders make the form overflow if we don't remove these 2px here
+      label
+        padding-inline-end textSpacing
+
+      & .form-control
+        color unset
+        font-size standardFontSize
+        padding-inline-start textSpacing
+        border none
+
+      &_buttons
+        margin-bottom 0
+        padding-bottom 0
+        text-align end
+        button:last-child, input:last-child
+          margin-inline-start standardSpacing
+
+    &__confirm
+      display flex
+      width 100%
+      justify-content flex-end
+      & > .iconbutton
+        margin-inline-start standardSpacing
+
+  &__columnAdder
```

Comment:
try to removed unused css rules
I know this one if for the add column button for kanban

Or this will create dead code that we might not dare to remove later

Reply:
Tried to remove all unused css rules, should be good by now